lib/install/phases/postsystem.go

```diff
@@ -67,10 +70,91 @@ type waitExecutor struct {
 	logrus.FieldLogger
 	// ExecutorParams is common executor params
 	fsm.ExecutorParams
+	// Client is the Kubernetes client
+	Client *kubernetes.Clientset
 }
 
 // Execute executes the wait phase
+// This waits for critical components to start within planet
 func (p *waitExecutor) Execute(ctx context.Context) error {
+	ctx, _ = context.WithTimeout(ctx, 5*time.Minute)
```

Comment:
nit: I don't think `cancel` should be suppressed but used to release whatever resources (goroutines or similar) if `waitForAPI` completes before the timeout:
```go
ctx, cancel := context.WithTimeout(ctx, 5*time.Minute)
defer cancel()
```